routers/zoo-router.js

```diff
@@ -0,0 +1,25 @@
+const router = require('express').Router();
+const knex = require('knex');
+
+const knexConfig = {
```

Comment:
Just make a knex file in the root directory and then you can just do `const knexConfig = require('../../knexfile');
const db = knex(knexConfig.development);` in your routes